configuration.yaml

```
@@ -1,10 +1,10 @@
 global:
   logDir: gs://otar000-evidence_input/parser_logs
   cacheDir: cache_dir
-  schema: https://raw.githubusercontent.com/opentargets/json_schema/2.3.2
-  EFOVersion: v3.54.0
+  schema: https://raw.githubusercontent.com/opentargets/json_schema/2.4.0
+  EFOVersion: v3.56.0
```

We are pinning 3.57 for this release
```suggestion
  EFOVersion: v3.57.0
```